Assignment-4/app/src/Components/TempConverter.js

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from "react";
+const TempConverter = () => {
+  const [tempCel, setTempCel] = useState(0);
+  const [tempFah, setTempFah] = useState(32);
+  const fohrenheitController = (e) => {
+    setTempFah(e.target.value);
```

Comment:
![image](https://github.com/kaushalyad321/ReactJs/assets/83696224/f1068642-9460-4828-b7e6-269b0458d6f9)
Giving incorrect result

Reply:
please check fixed the issues